elastic-agent-client.proto

```diff
@@ -268,14 +273,14 @@ message CheckinObserved {
     // one of the values have changed.
     optional CheckinObservedVersionInfo version_info = 3;
 
-    // Features are the expected feature flags configurations.
-    // Added on Elastic Agent v8.7.1.
-    Features features = 4;
```

Comment:
This deletion is backwards compatible, it isn't even sent in the messaging today.